k9mail/src/main/java/com/fsck/k9/activity/RecipientView.java

```diff
@@ -0,0 +1,296 @@
+package com.fsck.k9.activity;
+
+
+import java.util.Arrays;
+import java.util.List;
+
+import android.animation.Animator;
+import android.animation.AnimatorListenerAdapter;
+import android.text.TextWatcher;
+import android.view.View;
+import android.view.View.OnClickListener;
+import android.view.View.OnFocusChangeListener;
+import android.widget.Toast;
+import android.widget.ViewAnimator;
+
+import com.fsck.k9.FontSizes;
+import com.fsck.k9.R;
+import com.fsck.k9.activity.RecipientPresenter.CryptoMode;
+import com.fsck.k9.view.RecipientSelectView;
+import com.fsck.k9.view.RecipientSelectView.Recipient;
+import com.fsck.k9.mail.Address;
+import com.fsck.k9.mail.Message.RecipientType;
+import com.tokenautocomplete.TokenCompleteTextView.TokenListener;
+
+
+public class RecipientView implements OnFocusChangeListener, OnClickListener {
```

Comment:
We should probably find another name so MVP views are not confused with Android views. Maybe `RecipientMvpView`?
Of course we could also make this an actual Android view and let the activity create the presenter and pass it to this view.